src/components/Feed/Feeds/UrdfFeed.tsx

```diff
@@ -83,17 +84,23 @@ const View: FC<Props> = ({ feed }) => {
     new ROS3D.UrdfClient({
       ros: ros,
       tfClient: tfClient,
-      path: `http://${IP}:${port}`,
+      path: `http://${IP}:${descriptionServerPort}`,
       rootObject: viewer.scene,
     })
-  }, [IP, baseLinkName, port, viewer])
+  }, [IP, baseLinkName, descriptionServerPort, viewer])
 
   return <StyledViewer id={id} ref={ref} />
 }
 
 export const UrdfFeed: FC<Props> = ({ feed }) => {
-  const connected = useSelector(state => state.ros.connected)
+  const [state] = useService(rosService)
   return (
-    <Grid>{connected ? <View feed={feed} /> : <NoFeed text="no urdf" />}</Grid>
+    <Grid>
```

Comment:
@IceSentry Ici, si on a pas d'URDF on serait pas mieux de simplement ne pas l'afficher pour ne pas prendre de l'espace sur l'écran?

Reply:
c'est écrit "no urdf" mais en réalité sa veux dire que le robot est pas connecté. Je peu pas vraiment détecté qu'il n'y a pas d'URDF

Reply:
J'ai écrit ca pour faciliter le debugage

Reply:
Je viens de remarquer, quelques lignes plus loin tu l'as corrigé à "not connected".